dipy/reconst/mapmri.py

```diff
@@ -159,6 +160,9 @@ def __init__(self,
             the tissue diffusivity that is used when dti_scale_estimation is
             set to False. The default is that of typical white matter
             D=0.7e-3 _[5].
+        cvxpy_solver : cvxpy solver name
```

Comment:
This is a string? 

I would change this to: 

    cvxpy_solver : str, optional
        cvxpy solver name. Optionally optimize the positivity constraint with a particular cvxpy solver. See http://www.cvxp for details. Default: ??? 

And then fill in the default choice (what is the default?).